doc/zmq.md

```diff
@@ -75,6 +77,15 @@ notification `-zmqpubhashtx` the topic is `hashtx` (no null
 terminator) and the body is the hexadecimal transaction hash (32
 bytes).
 
+For wallet transaction notifications (both hash and tx), the
+topic also indicate if the transaction came from a block
+or mempool. If originated from mempool `-mempool` postfix
+will be added to the topic, for block `-block` postfix will
+be added. Because zeromq is using prefix matching for topics
+you can subscribe to `rawwallettx` (or `hashwallettx`) to get
+both notifications. If you only want one type of notification
+subscribe to either `rawwallettx-mempool` or `rawwallettx-block`.
```

Comment:
No documentation on how to determine *which* wallet this transaction involves (is it even possible?)

Reply:
It was actually submitted before the multi-wallet was merged. Wallet name (or identifier) can be part of the topic.